src/internal/m365/collection/drive/limiter_test.go

```diff
@@ -98,14 +77,10 @@ func (suite *LimiterUnitSuite) TestGet_PreviewLimits() {
 			enumerator: mock.EnumerateItemsDeltaByDrive{
 				DrivePagers: map[string]*mock.DriveItemsDeltaPager{
 					idx(drive, 1): {
-						Pages: []mock.NextPage{{
-							Items: []models.DriveItemable{
-								driveRootItem(rootID), // will be present, not needed
-								driveItemWithSize(idx(file, 1), namex(file, 1), parent(1), rootID, 7, isFile),
-								driveItemWithSize(idx(file, 2), namex(file, 2), parent(1), rootID, 1, isFile),
-								driveItemWithSize(idx(file, 3), namex(file, 3), parent(1), rootID, 1, isFile),
-							},
-						}},
+						Pages: pagesOf(rootAnd(
```

Comment:
I'd really prefer test structure updates were done in separate PRs. When test structure updates are mixed with other updates it becomes much more difficult to see if the test assertions were changed in some unexpected way

Reply:
You're right.  I should have them separated.  I believe I have refactors/changes basically discrete from here on out.  But feel free to ping me to split the PR if it's getting too rough to review.